---
description: Thank you all for joining the meeting.
---

# Planning page

How to pul the "\_raw.fastq.gz" files from the OSDR.

I think we used our time efficiently and would ask that you review (and continue to edit) the Doc notes from yesterday which are here:&#x20;

{% embed url="https://docs.google.com/document/d/10OFEDtOlVqtniyifgdmHB0D8HRb95u9KH68ok2wW77M/edit?usp=sharing" %}

\
We should check out the NMDC workflows?

{% embed url="https://nmdc-workflow-documentation.readthedocs.io/en/latest/chapters/3_MetaGAssemly_index.html" %}

[Link to edit planning spread sheet](https://docs.google.com/spreadsheets/d/1hjyx9FITrEW3UWRgjLmkzSKVCTmypf5Cg\_Pa5j8sIKk/edit?usp=sharing)



{% embed url="https://docs.google.com/spreadsheets/d/1hjyx9FITrEW3UWRgjLmkzSKVCTmypf5Cg_Pa5j8sIKk/edit?usp=sharing" %}

\
\- I think this could be automated (or done more elegantly than my copy-paste), at least for the OSDR records. That is what #4 below is doing, right @Richard? Then do it by hand for studies not in ALDS? - If we can automate this, awesome. Otherwise, there are more identified datasets (in the OSDR alone) than us. How should we approach it?

\
4\. Example Google colab notebook for pulling metadata for GLDS-69 â€¦ now evolving&#x20;



{% @github-files/github-code-block url="https://github.com/dr-richard-barker/Microbiome_seedlings_in_space/blob/main/Google_colab/OSDR_API_meta-transcriptome_microbiome_analysis.ipynb" %}



The Google Doc of our meeting notes and the Excel sheet are in this Google Drive folder:

{% embed url="https://drive.google.com/drive/folders/10wQXyi5sAjnAyaoCUqeAhAKsHQFFJpJH?usp=sharing" %}

Please feel free to drop anything else there.&#x20;

Explore "Bacdrive" resources for new insights:

{% embed url="https://bacdive.dsmz.de/" %}

If our efforts need a different approach for documents than an open-shared Google, let me know and I will work to set it up.\
\
