# References

Choi WG, Barker RJ, Kim SH, Swanson SJ, Gilroy S. (2019). Variation in the transcriptome of different ecotypes of Arabidopsis thaliana reveals signatures of oxidative stress in plant responses to spaceflight. American Journal of Botany, 106(1), 123-136. doi: 10.1002/ajb2.1223

Ewels P, Peltzer A, Fillinger S, Patel H, Alneberg J, Wilm A, Garcia MU, Di Tommaso P, Nahnsen S. (2020). The nf-core framework for community-curated bioinformatics pipelines. Nature Biotechnology. doi: 10.1038/s41587-020-0439-x

Segata N, Waldron L, Ballarini A, Narasimhan V, Jousson O, Huttenhower C. (2012). Metagenomic microbial community profiling using unique clade-specific marker genes. Nature Methods, 9(8), 811-814. doi: 10.1038/nmeth.2066

Leggett RM, Ramirez-Gonzalez RH, Clavijo BJ, Waite D, Davey RP. (2013). Sequencing quality assessment tools to enable data-driven informatics for high throughput genomics. Frontiers in Genetics, 4:288. doi: 10.3389/fgene.2013.00288

Ewels P, Magnusson M, Lundin S, Käller M. (2016). MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics (Oxford, England), 32(19), 3047-3048. doi: 10.1093/bioinformatics/btw354

Olanrewaju GO, Haveman NJ, Naldrett MJ, Paul AL, Ferl RJ, Wyatt SE. (2023). Integrative Transcriptomics and Proteomics Profiling of Arabidopsis thaliana Elucidates Novel Mechanisms Underlying Spaceflight Adaptation. Frontiers in Plant Science. doi: 10.3389/fpls.2023.1260429

Kruse CPS, Meyers AD, Basu P, Hutchinson S, Luesse DR, Wyatt SE. (2020). Spaceflight induces novel regulatory responses in Arabidopsis as revealed by combined proteomics and transcriptomic analyses. BMC Plant Biology, 20:237. doi: 10.1186/s12870-020-02392-6

Kruse CPS, Basu P, Luesse DR, Wyatt SE. (2017). Transcriptome and Proteome Stability in RNAlater® Preserved Tissue of Arabidopsis thaliana. PLOS One, 12(7): e0175943. doi: 10.1371/journal.pone.0175943

Paul AL, Sng NJ, Zupanska AK, Krishnamurthy A, Schultz ER, Ferl RJ. (2017). Genetic dissection of the Arabidopsis spaceflight transcriptome: Are some responses dispensable for the physiological adaptation of plants to spaceflight? PLoS ONE. doi: 10.1371/journal.pone.0180186

Zhou M, Sng NJ, LeFrois CE, Paul L, Ferl RJ. (2019). Epigenomics in an extraterrestrial environment: Organ-specific alteration of DNA methylation and gene expression elicited by spaceflight in Arabidopsis thaliana. BMC Genomics, 20(1). doi: 10.1186/s12864-019-5554-z

Totsline N, Kniel KE, Sabagyanam C, Bais HP. (2024). Simulated microgravity facilitates stomatal ingression by Salmonella in lettuce and suppresses a biocontrol agent. Scientific Reports, 14(1). doi: 10.1038/s41598-024-51573-y

Totsline N, et al. (2023). Microgravity and evasion of plant innate immunity by human bacterial pathogens. npj Microgravity. DOI: 10.1038/s41526-023-00323-x

Dixit AR, Khodadad CL, Hummerick ME, Spern CJ, Spencer LE, Fischer JA, Curry AB, Gooden JL, Maldonado Vazquez GJ, Wheeler RM, Massa GD, Romeyn MW. (2021). Persistence of Escherichia coli in the microbiomes of red Romaine lettuce (Lactuca sativa cv. ‘Outredgeous’) and mizuna mustard (Brassica rapa var. Japonica) - does seed sanitization matter? BMC Microbiology, 21(1). doi: 10.1186/s12866-021-02345-5

Kong D, Ye Z, Dai M, Ma B, Tan X. (2024). Light Intensity Modulates the Functional Composition of Leaf Metabolite Groups and Phyllosphere Prokaryotic Community in Garden Lettuce (Lactuca sativa L.) Plants at the Vegetative Stage. International Journal of Molecular Sciences, 25(3). doi: 10.3390/ijms25031451

Flores-Olivas LA, et al. (2020). Prevalence of Malassezia Species on the Skin of HIV-Infected Patients and Healthy Controls in Cali, Colombia. Nature, 586(7831), 661-665. Accessed October 15, 2023. Note: Because this reference was accessed online, you'll need to add "Accessed" followed by the date you retrieved it.

Seo DS, et al. (2021). Skin Commensal Fungus Malassezia and Its Lipases. Journal of Microbiology and Biotechnology, 31(5), 700-707.

Xu J, et al. (2017). Host–Microbe Interactions: Malassezia and Human Skin. PLoS Pathogens, 13(11), e1006696.

Cros, M. P., Mir-Pedrol, J., Toloza, L., Knödlseder, N., Maruotti, J., Zouboulis, C. C., Güell, M., & Fábrega, J. (2023). New insights into the role of Cutibacterium acnes-derived extracellular vesicles in inflammatory skin disorders. Scientific Reports, 13.[ https://doi.org/10.1038/s41598-023-43354-w](https://doi.org/10.1038/s41598-023-43354-w) Note: For online references without a specific publication date or publisher information, include the URL in the citation but omit it from the reference list.

Park, J., Onizuka, S., Seki, M., Suzuki, Y., Iwata, T., & Nakai, K. (2019). A systematic sequencing-based approach for microbial contaminant detection and functional inference. BMC Biology, 17(1).[ https://doi.org/10.1186/s12915-019-0690-0](https://doi.org/10.1186/s12915-019-0690-0)

Barker, R., Kruse, C.P.S., Johnson, C. et al. (2023). Meta-analysis of the space flight and microgravity response of the Arabidopsis plant transcriptome. npj Microgravity 9, 21.[ https://doi.org/10.1038/s41526-023-00247-6](https://doi.org/10.1038/s41526-023-00247-6)

Basavaraju, M., and B.S. Gunashree. 2023. ‘Escherichia Coli: An Overview of Main Characteristics’. Escherichia Coli - Old and New Insights. IntechOpen. doi:10.5772/intechopen.105508 Note: For books with multiple authors but no editor, list the authors' last names and initials followed by the year of publication, title, publisher information, and DOI (if available).

Savageau, M. A. (1983). Escherichia coli Habitats, Cell Types, and Molecular Mechanisms of Gene Control. The American Naturalist.[ https://doi.org/10.1086/284168](https://doi.org/10.1086/284168)

Ewels, P. A., et al. (2020). The nf-core framework for community-curated bioinformatics pipelines. Nature Biotechnology, 38(3), 276-278.

Andrews, S. (2010). FastQC: a quality control tool for high throughput sequence data. \[Software application]. Available from https://www.bioinformatics.babraham.ac.uk/projects/fastqc/

Ewels, P., et al. (2016). MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics, 32(19), 3047-3048.

Krueger, F. (2015). Trim Galore! \[Software application]. Available from[ https://github.com/FelixKrueger/TrimGalore](https://github.com/FelixKrueger/TrimGalore)

Bushnell, B. (2014). BBMap: A Fast, Accurate, Splice-Aware Aligner. \[Software application]. Available from[ https://sourceforge.net/projects/bbmap/](https://sourceforge.net/projects/bbmap/)

Li, H. (2013). Seqtk: Toolkit for processing sequences in FASTA/Q formats. \[Software application]. Available from[ https://github.com/lh3/seqtk](https://github.com/lh3/seqtk)

Bankevich, A., et al. (2012). SPAdes: A high-performance, short-read assembler. Journal of Computational Biology, 19(5), 455-477.

Li, D., et al. (2015). RSEM: accurate transcript quantification from RNA-Seq data. Bioinformatics, 31(10), 1674-1676.

Haas, B. J., et al. (2013). Genome sequence determination, assembly and annotation of Polynucleobacter necessarius type strain ATCC 43124. Nature Protocols, 8(8), 1494-1512.

Seemann, T. (2014). Prokka: rapid prokaryotic genome annotation. Bioinformatics, 30(14), 2068-2069.

Hyatt, D., et al. (2010). DIAMOND: a sensitive algorithm for short read similarity search. BMC Bioinformatics, 11(1), 119.

de Zelicourt, A. et al. (2013). Rhizosphere Microbes as Essential Partners for Plant Stress Tolerance. Molecular Plant, 6(2), 242-245.

Fu, Y. et al. (2016). How to Establish a Bioregenerative Life Support System for Long-Term Crewed Missions to the Moon or Mars. Astrobiology, 16(12), 925-936.

Paul, A.L. et al. (2013). Spaceflight Transcriptomes: Unique Responses to a Novel Environment. Astrobiology, 13(5), 456-475.

Pieterse, C.M. et al. (2014). Induced Systemic Resistance by Beneficial Microbes. Annual Review of Phytopathology, 52, 347-375.

Kim, Y.J. et al. (2020). Weizmannia ginsengihumi sp. nov., Isolated from Soil of a Ginseng Field. International Journal of Systematic and Evolutionary Microbiology, 70(3), 1884-1889.

\












