---
description: >-
  The analysis does reveal a "plant defence response" but no microbes were
  visible on the plants, so how were the microbes inside the plant responding?
---

# Transcriptional\_analysis

The analysis does reveal a "plant defence response" but no microbes were visible on the plants, so how were the microbes inside the plant responding?

Below is a link to a github repo with a google colab contain python script that can pull data from our OSD-#'s of interest. [OSDR\_API\_meta-transcriptome\_microbiome\_analysis.ipynb](../Google\_colab/OSDR\_API\_meta-transcriptome\_microbiome\_analysis.ipynb)&#x20;

A few files used for transcriptional assessment of these OSDR samples. This metadata file contains the factors for all the processed plant studies currently available through the OSD API.&#x20;

#### [Matrix\_6Factors\_38,120,37,321,217.csv](Matrix\_6Factors\_38%2C120%2C37%2C321%2C217.csv)

#### Counts file that pairs with 6 Factor table.

#### [MARIX5\_OSD\_37\_38\_120\_217\_321\_merged\_counts.csv.zip](MARIX5\_OSD\_37\_38\_120\_217\_321\_merged\_counts.csv.zip)

#### Differential gene expression results from the Fl vs GC model.&#x20;

#### [FL\_vs\_GC\_OSD-37OSD-38OSD-120OSD-217OSD-321\_DGE.csv](FL\_vs\_GC\_OSD-37OSD-38OSD-120OSD-217OSD-321\_DGE.csv)

Future work could be to do Genome ontology analysis to further disect this broadly detected genetics response to spaceflight.&#x20;



{% @github-files/github-code-block url="https://github.com/dr-richard-barker/Microbiome_of_seedlings_in_space/tree/main/Transcriptional_analysis" %}
